### ISSUE TITLE
no-console should handle console imported from `node:console`

### ISSUE THREAD

**Issue by dsherret** [denoland/deno_lint#1316]
When `no-console` is enabled, the following should raise a diagnostic:

```ts
import console from "node:console";

console.log("hi");
```
